src/gui/guiFormSpecMenu.cpp

```diff
@@ -2091,7 +2088,7 @@ void GUIFormSpecMenu::parseTabHeader(parserData* data, const std::string &elemen
 		}
 
 		auto style = getStyleForElement("tabheader", name);
-		e->setNotClipped(style.getBool(StyleSpec::NOCLIP, data->current_parent == this));
+		e->setNotClipped(style.getBool(StyleSpec::NOCLIP, true));
```

Comment:
I someone really wants to use a tabheader element in a `scroll_container`, they should step it clipped themselves.